frontend/src/App.js

```diff
@@ -46,6 +41,23 @@ class App extends Component {
     });
   };
 
+  // Logout state handler function
+  logout = () => {
+    this.setState({
+      sessionId: undefined,
+      showShifts: false
+    });
+  };
+
+  // Get user attributes
+  fetchUserAttributes = sessionId => {
+    return this.fetchRequest(
+      "http://localhost:3000/users/me/",
```

Comment:
You can keep the base url a constant within your handler, and just pass the endpoint (`users/me`)